prisma/schema.prisma

```diff
@@ -11,12 +11,174 @@ datasource db {
 }
 
 model User {
-  username   String @id
-  password   String
-  nama_depan String
+  username      String   @id
+  password      String
+  nama_depan    String
   nama_belakang String
-  alamat String
-  no_hp String
+  alamat        String
+  no_hp         String
   tanggal_lahir DateTime
-  peran String
+  peran         String
+
+  tim_manajer   Tim?
+  pemain        Pemain?
+  wasit         Wasit?
+  pelatih       Pelatih?
+  penonton      Pembelian_Tiket[]
+  peminjamans   Peminjaman[]
+  panitia       Rapat[]           @relation("panitia")
+  manajer_tim_A Rapat[]           @relation("manajer_tim_A")
+  manajer_tim_B Rapat[]           @relation("manajer_tim_B")
+  undangan      Undangan[]
+  pendaftarans  Pendaftaran[]
+  ujians        Ujian[]
+}
+
+model Tim {
+  nama             String @id
+  universitas      String
+  username_manajer String @unique
+  manajer          User   @relation(fields: [username_manajer], references: [username])
+
+  pemains       Pemain[]
+  pelatihs      Pelatih[]
+  pertandingans Tim_Pertandigan[]
+  undangans     Undangan[]
+  pendaftarans  Pendaftaran[]
+}
+
+model Pemain {
+  username String @id
+  user     User   @relation(fields: [username], references: [username])
+
+  nama_tim String? @unique
```

Comment:
please revise. no need unique constraint

Reply:
i agree

Reply:
thanks for the input